.code-samples.meilisearch.yaml

```diff
@@ -4,25 +4,25 @@
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
 get_one_index_1: |-
-  $client->getIndex('movies');
+  $client->index('movies')->getRaw();
```

Comment:
Shouldn't this use `fetchInfo()` instead of `getRaw()`, returning an Index instead of an http response?

Reply:
we chose to let the raw option for the moment -> https://github.com/meilisearch/meilisearch-js/pull/715/files#r539555931

Reply:
Ok, then everything LGTM 🎉